package.json

```diff
@@ -18,7 +18,7 @@
     "typecheck:watch": "yarn run tsc -w --noEmit"
   },
   "engines": {
-    "node": ">=14.15.2 <15.0.0"
+    "node": "^14.17.0 <15.0.0"
```

Comment:
Ah, going along with the previous comment about node versions, perhaps if this is happening in the `package.json`, this is also the time to reconcile the difference in the browser, the dockerfile (`browser.dockerfile`  on `14.15.2`) and the browser CI (`browser-ci.yml`, on `14.19.3`).

Or at the very least, I think, we should file an issue to come and look at keeping these node versions in sync.

Reply:
I'll change it by hand for now, but what we should really be doing is figuring out how to eliminate the need to keep them in sync. We'd have to figure out if there's a reliable way to map from the version given in `engines` to the proper Dockerhub package.

Reply:
But at least we could probably sync the version up automatically between `package.json` and the two `ci.yml` files.